forge/src/main/java/com/teamresourceful/resourcefulbees/client/gui/screen/FakeFlowerScreen.java

```diff
@@ -0,0 +1,43 @@
+package com.teamresourceful.resourcefulbees.client.gui.screen;
+
+import com.mojang.blaze3d.vertex.PoseStack;
+import com.teamresourceful.resourcefulbees.ResourcefulBees;
+import com.teamresourceful.resourcefulbees.client.components.BeeconEffectWidget;
+import com.teamresourceful.resourcefulbees.common.blockentity.FakeFlowerEntity;
+import com.teamresourceful.resourcefulbees.common.inventory.menus.FakeFlowerMenu;
+import com.teamresourceful.resourcefullib.client.utils.RenderUtils;
+import net.minecraft.client.gui.screens.inventory.AbstractContainerScreen;
+import net.minecraft.network.chat.Component;
+import net.minecraft.resources.ResourceLocation;
+import net.minecraft.world.entity.player.Inventory;
+import org.jetbrains.annotations.NotNull;
+
+public class FakeFlowerScreen extends AbstractContainerScreen<FakeFlowerMenu> {
+
+    private final FakeFlowerEntity tileEntity;
+    public FakeFlowerScreen(FakeFlowerMenu container, Inventory inventory, Component displayName) {
+        super(container, inventory, displayName);
+        tileEntity = container.getEntity();
```

Comment:
Is there a reason for the tileentity because all I see is the != null below which doesnt really make much sense.